spec/graphql/mutations/users/create_user_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+module Mutations
+  module Users
+    RSpec.describe CreateUser, type: :request do
+      describe 'create user' do
+        def query
+          <<~GQL
+            mutation {
+              createUser(input:
+                { username: "Lassie", email: "sillyboy@inwell.com" } ) {
+      	      user {
```

Comment:
⛏️ Love the use of actual GQL! Generated GQL can be harder to use. I might format the tabbing here so it is apparent which curly brackets go with each other!